eth/ethereum.go

```diff
@@ -130,7 +130,7 @@ func (c *EthereumClient) EthAddress() (*ethCommon.Address, error) {
 // EthSuggestGasPrice retrieves the currently suggested gas price to allow a
 // timely execution of a transaction.
 func (c *EthereumClient) EthSuggestGasPrice(ctx context.Context) (*big.Int, error) {
-	return c.client.SuggestGasPrice(ctx)
+	return c.client.SuggestGasTipCap(ctx)
```

Comment:
What about BaseFee? Based on other tests I've made we need to sum basefee plus the gas tip.